src/TSTransformer/macros/propertyCallMacros.ts

```diff
@@ -454,7 +455,14 @@ const READONLY_ARRAY_METHODS: MacroList<PropertyCallMacro> = {
 				start: lua.number(1),
 				end: size,
 				step: lua.number(1),
```

Comment:
Defaults to 1 in lua. Necessary?

Reply:
start and step are both required and the 1 is needed because the argument isn't optional.

Reply:
Step is not required. I think it looks cleaner to leave it undefined.